weave/integrations/langchain/langchain.py

```diff
@@ -188,9 +184,12 @@ def _persist_run_single(self, run: Run) -> None:
                             # of an entire Parent object.
                             parent_run = self.gc.call(wv_current_run.parent_id)
 
+            fn_name = make_pythonic_function_name(run.name)
+            complete_op_name = f"langchain.{run.run_type.capitalize()}.{fn_name}"
+            complete_op_name = truncate_op_name(complete_op_name)
             call = self.gc.create_call(
```

Comment:
we do use this pattern in the llamaindex integration too, but they have hardcoded "op" names which are all reasonably sized. We are relying on them staying well behaved, but because they probably will be this is just a nit.

Reply:
👍 agreed